test/cypress/integration/settings/your-psp-stripe-tasklist.cy.test.js

```diff
@@ -192,4 +198,30 @@ describe('Your PSP Stripe page', () => {
       cy.get('strong[id="task-Company-number-status"]').should('contain', 'complete')
     })
   })
+
+  describe('Service director task', () => {
+    it('should click on service director task and redirect back to tasklist when valid service director information is submitted', () => {
+      setupYourPspStubs()
+
+      cy.get('span').contains('Service director').click()
+      cy.get('h1').should('contain', 'Enter a director’s details')
+      cy.get('#first-name').type(typedFirstName)
+      cy.get('#last-name').type(typedLastName)
+      cy.get('#dob-day').type(typedDobDay)
+      cy.get('#dob-month').type(typedDobMonth)
+      cy.get('#dob-year').type(typedDobYear)
+      cy.get('#email').type(typedEmail)
+      cy.get('#director-form > button').click()
```

Comment:
We're missing an assertion that after clicking the submit button we are redirected to the task list page.